content/en/agent/logs/proxy.md

```diff
@@ -60,17 +63,236 @@ The parameter above can also be set with the following environment variable:
 * `DD_LOGS_CONFIG_SOCKS5_PROXY_ADDRESS`
 
 {{% /tab %}}
-{{% tab "HTTPS" %}}
+{{< /tabs >}}
 
-When the Agent is [configured to send logs through HTTPS][1], use the same [set of proxy settings][2] as the other data types in order to send logs through a web proxy.
+## Examples of TCP proxy
 
-[1]: /agent/logs/#send-logs-over-https
-[2]: /agent/proxy/
-{{% /tab %}}
-{{< /tabs >}}
+### Using HAProxy as a TCP Proxy for Logs
+
```

Comment:
So I would actually start with the Agent configuration.

As it is a shorter part, it might be more impactful to have it first, what do you think?

Reply:
+1 to this idea, moving up the agent config section will help for engagement

Reply:
done